NewForce-Cohort-4/tabloid-full-stack-team-fuschia
# Dtb edit post

# Description
As an author I would like to be able to modify my Posts so that I can correct mistakes or add additional content.

**Given** the user is viewing the Post list
**When** they select the option to edit an Post
**Then** the user should be directed to a form and given the ability to change the Post's information

**Given** the user is finished updating the Post information
**When** they click the Save button
**Then** the updated Post should be saved to the database
**And** the user should be redirected to the new Post's details page

**Given** the user has decided not to edit the Post
**When** they click the Cancel button
**Then** the user should be redirected back to the list page

> **Editable Information**
> Title
> Content
> Category
> Header Image URL (optional)
> Publication Date (optional)

## Type of change
Please delete options that are not relevant.

# Testing Instructions
- pull down new branch
- login
- select "New Post" from the Nav
- create a new post - you should be redirected to the detail page
- Go to "My Posts"
- click "Edit" on your newly created Post
- you should be navigated to the detail page. Check to make sure that the edits appear as expected.

# Checklist:
